astropy/stats/funcs.py

```diff
@@ -577,20 +586,22 @@ def poisson_conf_interval(n, interval='root-n', sigma=1, background=0,
        C = \left[ \int_0^\infty \frac{e^{-(S+B)}(S+B)^N}{N!} dS \right] ^{-1}
        = \left( \sum^N_{n=0} \frac{e^{-B}B^n}{n!}  \right)^{-1}
 
-    See [KraftBurrowsNousek][kbn1991] for further details.
+    See `Kraft, Burrows, and Nousek (1991)
+    <http://adsabs.harvard.edu/abs/1991ApJ...374..344K>`_ for further
+    details.
 
     These formulas implement a positive, uniform prior.
-    [KraftBurrowsNousek][kbn1991] discuss this choice in more detail and show
-    that the problem is relatively insensitive to the choice of prior.
+    `Kraft, Burrows, and Nousek (1991)
+    <http://adsabs.harvard.edu/abs/1991ApJ...374..344K>`_ discuss this
+    choice in more detail and show that the problem is relatively
+    insensitive to the choice of prior.
 
-    This functions has an optional dependency: Either scipy or
-    `mpmath <http://mpmath.org/>`_  need to be available. (Scipy only works for
+    This functions has an optional dependency: Either scipy or `mpmath
```

Comment:
Can Sphinx understand line break in the middle of the linking like this?

Reply:
It seems to.  The docs rendered fine locally.

I just pushed a change to fix typos I noticed in this line. :)